R/git.R

```diff
@@ -71,12 +71,10 @@ use_git_hook <- function(hook, script) {
     stop("This project doesn't use git", call. = FALSE)
   }
 
-  base_path <- git2r::discover_repository(proj_get())
   use_directory(".git/hooks")
-
   hook_path <- file.path(".git/hooks", hook)
-  write_over(base_path, hook_path, script)
```

Comment:
This made me realise usethis should have a `proj_path()` function for constructing paths in the current project